Backend/Remora.Discord.Gateway/BaseGatewayClient.cs

```diff
@@ -454,7 +455,7 @@ private async Task<Result> GatewayReceiverAsync(CancellationToken ct)
 
             while (!ct.IsCancellationRequested)
             {
-                var receiveResult = await this.TransportService.ReceivePayloadAsync(ct);
+                var receiveResult = await this.TransportService.ReceivePayloadAsync(ct).ConfigureAwait(false);
```

Comment:
Any reason for the `CA(false)` here?

Reply:
This is a call that will be waited on an awful lot, so I've followed the guidelines laid out [here](https://devblogs.microsoft.com/dotnet/configureawait-faq/) - this is contained code that should have no need to be invoked on a particular scheduler and I'd like to avoid the associated overhead.